wordpress-mobile/WordPress-iOS
# Issue/15165 xcode12 misc build warnings

Fixes #15165

To test:

1. Build and make sure that none of the warnings listed in #15165 are not there anymore.
2. tap the FAB then Site page
3. Make sure that the layout picker and the page creation experience does not differ from `develop`
4. tap the FAB then Story Post
5. Make sure that the introductory page shows the two images 'Patagonia' and 'Hiking in the Southwest'
6. Go to the reader and make sure you can follow/unfollow sites as expected

PR submission checklist:
